server.py

```diff
@@ -108,11 +108,32 @@ def __init__(self, map_name):
                     await client.send_json(self.state.as_dict(map_name))
             return ws
 
+
+    async def play_round(self, state):
+        all_selected = True
+        for robot in self.state.robots:
```

Comment:
Nerozumím těm dvěma ifům v tomto cyklu. Pokud v programu robotů bývá None, pak se nestane, že program je kratší než 5. Pokud bývá program kratší než 5, nebudou v něm None?

Reply:
Tohle se musí ještě vyřešit a bude se to upravovat. A ty určitě přijdeš na to snadněji, jestli tam zatím nechat oba a nebo který z nich.
Problém je v tom, že robot.program v backendu je na začátku prázdný seznam, proto ta podmínka < 5,  tedy není tam None.
Kdežto až potom když klient vybírá ty karty, tak se přiřazují ty karty do programu, včetně None.
Když jsme na srazu to zkoušeli, tak to právě nefungovalo pouze s tou podmínkou `if None in robot.program:` protože tam byl ten prázdný seznam.
@encukou co je teď nejlepší varianta, poradíš prosím?

Reply:
Na selský rozum bych změnila robotí program, aby obsahoval 5x None, a nebudeme muset řešit dva rozdílné stavy.

Reply:
To stejné mě taky napadlo. 😁